acousticbrainz/views/index.py

```diff
@@ -7,6 +7,8 @@
 @index_bp.route("/")
 def index():
     stats, last_collected = get_stats()
+    for key in stats:
+        stats[key] = "{:,}".format(stats[key])
```

Comment:
Is this the standard jinja way to format this? e.g. django has localisation in the templating itself, https://docs.djangoproject.com/en/1.8/topics/i18n/formatting/, which lets us use different separators depending on selected language